### ISSUE TITLE
A11y_Visual studio Code Client_Editor_More actions_Non Text Contrast: Focus is not clearly visible on "more actions" and items present inside "More actions" button ratio is 1.48:1 less than required

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#30890]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
 -A11yTCS; -A11ySev2; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2023; -WCAG1.1.1; -Non Text Content; 

**Environment and OS details:**
Visual Studio Code 
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992

**Repro Steps:**
Open Visual Studio Code 
Tab till More button present in editor section.
Select More actions button. 
Verify whether Focus is not clearly visible on "more actions" and items present inside "More actions" button or not.

**Actual Result:**
Focus is not clearly visible on "more actions" and items present inside "More actions" for example "Show open Editors", "Close All", "Close saved" etc. buttons ratio is 1.48:1 less than required 3:1.

**Expected Result:**
Focus should be clearly visible on "more actions" and items present inside "More actions" for example "Show open Editors", "Close All", "Close saved" etc. buttons ratio should be 3:1 or greater than 3:1.

**Attachment:**
<img width="942" alt="External to VS Code" src="https://github-com/microsoft/vscode/assets/98530069/40ef90cb-9f5a-4628-96b9-dfecad0bac61">
